WiinUPro/Windows/MainWindow.xaml

```diff
@@ -72,11 +72,13 @@
-                            <CheckBox x:Name="settingAutoAddXInputDevices" Content="Auto Add XInput Devices" HorizontalAlignment="Left" Margin="10,110,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingAutoAddXInputDevices_Checked" ToolTip="Automatically connect emulated XInput Devices based on device rumble configurations."/>
-                            <CheckBox x:Name="settingMinimizeOnClose" Content="Minimize To System Tray" HorizontalAlignment="Left" Margin="10,130,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingMinimizeOnClose_Checked" ToolTip="Closing the application will instead minimize it to the system tray."/>
+                            <CheckBox x:Name="settingExclusiveMode" Content="Use Exclusive Mode" HorizontalAlignment="Left" Margin="10,70,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingExclusiveMode_Checked"/>
+                            <CheckBox x:Name="settingToshibaMode" Content="Use Toshiba Mode" HorizontalAlignment="Left" Margin="10,90,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingToshibaMode_Checked"/>
+                            <CheckBox x:Name="settingSuppressLostConn" Content="Hide Connection Errors" HorizontalAlignment="Left" Margin="10,110,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingSuppressLostConn_Checked"/>
+                            <CheckBox x:Name="settingAutoAddXInputDevices" Content="Auto Add XInput Devices" HorizontalAlignment="Left" Margin="10,130,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingAutoAddXInputDevices_Checked" ToolTip="Automatically connect emulated XInput Devices based on device rumble configurations."/>
+                            <CheckBox x:Name="settingMinimizeOnClose" Content="Minimize To System Tray" HorizontalAlignment="Left" Margin="10,50,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingMinimizeOnClose_Checked" ToolTip="Closing the application will instead minimize it to the system tray."/>
+                            <CheckBox x:Name="settingProfileQueuing" Content="Profile Queuing" HorizontalAlignment="Left" Margin="10,150,0,0" VerticalAlignment="Top" Style="{StaticResource CheckBoxStyle}" Click="settingProfileQueuing_Checked" IsEnabled="False"
+                                      ToolTip="Profiles whose names end with &quot;_№&quot; (for example &quot;Nunchuk_Xinput_IR_1.wup&quot;) will be automatically switched between depending on the currently connected devices.&#x0a;The number should correspond with the selected rumble controller of the profile. This functionality is only available for Nintendo controllers."/>

```

Comment:
The tooltip is now on a separate line, although it could use some reformulation.